pkg/webui/components/submit-button/index.js

```diff
@@ -33,17 +32,16 @@ class SubmitButton extends React.PureComponent {
     icon: undefined,
   }
   render() {
-    const { message, icon, isSubmitting, isValidating, disabled, dirty } = this.props
+    const { message, icon, disabled, isSubmitting, isValidating } = this.props
 
-    const buttonDisabled = disabled || isSubmitting || !dirty
```

Comment:
```js
const buttonDisabled = disabled || buttonLoading
```

We do want to disable the submit button when the form is submitting/validating and the button is in the loading state, otherwise the user can still send requests which can result in weird behavior, especially with slow internet connection. Also, I dont see any reasons why should we allow the submit button to be clickable when the form is submitting.

Reply:
The `<Button />` is de-facto disabled, when it is set to `busy`:
https://github.com/TheThingsNetwork/lorawan-stack/blob/develop/pkg/webui/components/button/index.js#L73-L75

Reply:
The `<Button />` is still clickable and since it has `type="submit"` the button submits the form data to the server on each click. Please try it yourself.

Throttle your network connection in the dev tools and try submitting the form, say application create` several times while the `<Button />` is in the `busy` state.

Reply:
Fixed this. The busy state will now also make the button disabled via the `disabled` prop and also disregard any click events that might still come through. I've also fixed a related issue in the oauth login form, where the button would be clickable again right before the redirect.